projects/plugins/jetpack/_inc/client/search-dashboard-entry.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import ReactDOM from 'react-dom';
+import React from 'react';
+import { Provider } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import store from 'state/redux-store';
+import SearchDashboard from './search/dashboard';
+
+/**
+ * Mounts the Search Dashboard to #jp-search-dashboard if available.
+ */
+function init() {
+	const container = document.getElementById( 'jp-search-dashboard' );
+
+	if ( container === null ) {
+		return;
+	}
+
+	ReactDOM.render(
+		<Provider store={ store }>
+			<SearchDashboard />
+		</Provider>,
+		container
+	);
+}
+
+// Initialize Instant Search when DOMContentLoaded is fired, or immediately if it already has been.
```

Comment:
Needs a comment update; this isn't the Instant Search application.

```suggestion
// Initialize the dashboard when DOMContentLoaded is fired, or immediately if it already has been.
```